src/client/components/app/LineDetailsDisplay/PositionsTable.tsx

```diff
@@ -118,14 +127,14 @@ export const PositionsTable = ({ positions, displayLine = false }: PositionsProp
     }
   }, [lineAddress, selectedLine]);
 
-  //Action Handlers for positions table
+  // Action Handlers for positions table
 
-  const depositHandler = (position?: string) => {
+  const depositHandler = (position?: string, proposal?: string) => {
     if (!userWallet) {
       connectWallet();
     } else {
-      console.log('accept Proposal', position);
       dispatch(LinesActions.setSelectedLinePosition({ position }));
+      dispatch(LinesActions.setSelectedLinePositionProposal({ proposal }));
```

Comment:
if `proposal` is optional arg then we should only dispatch action if its non-null no?

Reply:
not going to change rn as it's doesn't effect any functionality and you could say the same about how `position` is being dispatched in legacy code